### ISSUE TITLE
Redis already declared

### ISSUE THREAD

**Issue by ekandreas** [weprovide/valet-plus#42]
Getting already declared error for Redis when installing Valet-plus `valet install`:
```
PHP Fatal error:  Cannot declare class Redis, because the name is already in use in /Users/aek/.composer/vendor/weprovide/valet-plus/cli/includes/facades.php on line 35
PHP Stack trace:
PHP   1. {main}() /Users/aek/.composer/vendor/weprovide/valet-plus/cli/valet.php:0
PHP   2. require() /Users/aek/.composer/vendor/weprovide/valet-plus/cli/valet.php:10
PHP   3. ComposerAutoloaderInit41a7cc1ebcee16e4d84f7173b844eaac::getLoader() /Users/aek/.composer/vendor/autoload.php:7
PHP   4. composerRequire41a7cc1ebcee16e4d84f7173b844eaac() /Users/aek/.composer/vendor/composer/autoload_real.php:56
```
Class Redis is already declared in cli/Valet/Redis.php so...
I have cleared cache and removed all global composer requirement except for valet-plus.

**Comment by 9IPHP** [comment]
I have the same issue. I had removed valet before install valet-plus.

**Comment by FrancisVarga** [comment]
i have the same problem here:
valet install
Password:

> PHP Fatal error:  Cannot declare class Redis, because the name is already in use in /Users/francisvarga/.composer/vendor/weprovide/valet-plus/cli/includes/facades.php on line 35
> 
> Fatal error: Cannot declare class Redis, because the name is already in use in /Users/francisvarga/.composer/vendor/weprovide/valet-plus/cli/includes/facades.php on line 35

**Comment by fridzema** [comment]
+1 Same problem here

**Comment by timneutkens** [comment]
Super weird, still can't reproduce.

**Comment by spaceemotion** [comment]
I'm using Mac OS sierra, using a fresh copy of valet-plus. wasn't able to use a single command after the install.

**Comment by deizel** [comment]
@timneutkens You can reproduce with `brew install php71-redis`, which adds a [`Redis` class](https://github.com/phpredis/phpredis#class-redis).

**Comment by timneutkens** [comment]
@deizel :joy: those namespaces haha, will check it out whenever I find the time 👍

**Comment by timneutkens** [comment]
Fixed in 1.0.2 https://github.com/weprovide/valet-plus/releases/tag/1.0.2
